Frameworks/UIKit/UIFont.mm

```diff
@@ -362,99 +65,35 @@ + (UIFont*)preferredFontForTextStyle:(NSString*)style {
     return StubReturn();
 }
 
+/**
+ @Status Stub
+*/
 + (UIFont*)fontWithData:(NSData*)data {
```

Comment:
This turns out to be a WinObjC-only function anyway - it's not something we have to support. It is used in UIApplicationMain and CGFont as what looks like a convenience method for their own purposes?

---
In reply to: [78222890](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78222890) [](ancestors = 78222890,78118792)